RELEASES.md

```diff
@@ -0,0 +1,52 @@
+# Release cycle and process
+Velero team is aiming for 3 releases per year with few patch releases in between.
```

Comment:
I love this, so first of thank you!  `aiming for 3 releases` indicates that there will be some sort of a notion of a timed release and if $feature is not done it will be moved to the next release.    This will be done to give project managers guidance on the when to expect a release which is helpful, so thank you.

I wonder if we could firm up the governance a bit with some flexibility.   What would folks think of the following language...

`Velero team is aiming for 3 releases per year, but will commit to two releases per year with few patch releases in between`  

Something along those lines to be more concrete. WDYT?

Reply:
Sounds good to me, that give some more room indeed!

Reply:
added few more lines about that